.github/workflows/daily-testing-build.yaml

```
@@ -47,6 +55,10 @@ jobs:
           token: ${{ secrets.PODMAN_DESKTOP_BOT_TOKEN }}
           fetch-depth: 0
 
+      - name: Set the default env. variables
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)